### DISCUSSION TITLE
Infinite uploads

### DISCUSSION THREAD

**Discussion by il-Bisat** [bootscore/bootscore#845] [General]
Hello everyone, I installed the latest version of the theme and I noticed that in the media I can no longer see the image grid, only the list. If I upload a new image I get a generic error and asks me to try again.

This behavior doesn't happen immediately, I don't know what it depends on, with all plugins deactivated it doesn't happen and not even with another theme or standard theme.

I noticed that when I reloaded the theme the anomaly disappeared and then returned.

I can't even load new plugins and in front I'm having problems adding items to the cart.

In the console from the media when I go to the grid view this error appears in the console with reference to the upload.php file:
jQuery(function ($) {
                var alert_w = $('#woof_alert_woobe');
                alert_w.on('click', '.notice-dismiss', function (e) {
                    $.post(ajaxurl, {
                        action: 'woof_dismiss_alert',
                        alert: 'woocommerce_bulk_editor',
                        sec: 0d25e0766b });
                });

woof should refer to an installed plugin, but even when I deactivate it I find the same anomaly.

I would like to know if anyone else has encountered similar problems or if you have any advice, thanks

**Comment by il-Bisat** [comment]
upload.php?mode=grid:724 Uncaught SyntaxError: Invalid or unexpected token (at upload.php?mode=grid:724:30)

This is the message on the console

**Comment by crftwrk** [comment]
Hi,

I'm pretty sure this has nothing to do with the theme because it doesn't add anything to the backend other than CSS to the Gutenberg editor. Means that have to be something in a plugin. So, you should deactivate **ALL** plugins as you did and then activate each step-by-step and checking the error each until you have found it.

However, it's hard to say something without having a link. You can create an admin account, send data via https://bootscore.me/private-info-form/ and post the key here. Then we can have a closer look to it.

**Reply by il-Bisat** [reply]
Thanks for the reply, yes that's what I'm doing.

However, I noticed the presence of this message in the error.log:
[27-Sep-2024 07:38:27 UTC] PHP Warning: PHP Startup: Unable to load dynamic library 'php_imagick.dll' (tried: C:/Users/User/AppData/Roaming/Local/lightning-services/php -8.1.23+0/bin/win64/ext\php_imagick.dll (The specified module could not be found), C:/Users/User/AppData/Roaming/Local/lightning-services/php-8.1.23+0/bin /win64/ext\php_php_imagick.dll.dll (The specified module could not be found)) in Unknown on line 0

I'm working locally, and it's the only site where I've encountered this anomaly.

**Reply by crftwrk** [reply]
> (The specified module could not be found)

The PHP module is missing on the server.

> I'm working locally, and it's the only site where I've encountered this anomaly.

Transfer your project to a another server and problem should be solved.
